module/Company/src/Service/Company.php

```diff
@@ -3,26 +3,34 @@
 namespace Company\Service;
 
 use Application\Service\FileStorage;
-use Company\Form\EditCategory;
-use Company\Form\EditCompany;
-use Company\Form\EditJob;
-use Company\Form\EditLabel;
-use Company\Form\EditPackage;
-use Company\Mapper\BannerPackage;
-use Company\Mapper\Category;
-use Company\Mapper\FeaturedPackage;
-use Company\Mapper\Label;
-use Company\Mapper\LabelAssignment;
-use Company\Mapper\Package;
-use Company\Model\Job;
-use Company\Model\Job as JobModel;
-use Company\Model\JobCategory as CategoryModel;
-use Company\Model\JobLabel as LabelModel;
-use Company\Model\JobLabelAssignment;
+use Company\Form\{
```

Comment:
By changing these imports, most of the getXxxForm methods now have incorrect PhpDoc comments. Please fix these. They are also reported by PhpStan